meta/documents/changelog_de.md

```diff
@@ -1,5 +1,10 @@
 # Release Notes für PAYONE
 
+## X.X.X
```

Comment:
What version is this?

Reply:
We had discussed once that if we just want to bring a change in a plugin without making a new version, that we add the changelog but do not adapt a version.

I am merging here into the base branch for the Klarna payment methods.
When they go live a version and another changelog has to be added.